README.md

```diff
@@ -49,3 +49,42 @@ const clientPlugin = new IlpPluginXrpAsymClient({
   claimInterval: 5 * 60 * 1000
 })
 ```
+
+# Interfaucet example
+
+Full example of obtaining money from the Interfaucet using this plugin:
+```js
+const Plugin = require('.')
+const crypto = require('crypto')
+const IlDcp = require('ilp-protocol-ildcp')
+const IlpPacket = require('ilp-packet')
+function sha256(preimage) { return crypto.createHash('sha256').update(preimage).digest() }
+
+const plugin = new Plugin({
+  xrpServer: 'wss://s.altnet.rippletest.net:51233',
+  secret: 'sspPGRjcBXT9UBxewQUJKcWZCR1zC',
```

Comment:
This may stop working if the testnet is reset. It might be better to get new credentials programmatically